src/pages/Litmus/Litmus.tsx

```diff
@@ -6,27 +6,30 @@ import "./Litmus.sass";
 const Litmus = (): JSX.Element => {
   return (
     <>
-      <h1
-        style={{ padding: "1.5rem 0" }}
-        className="spaced-text capitalized-text"
-      >
+      <h1 className="spaced-text capitalized-text">
         Financial Calculator
       </h1>
 
       <div className="primary-details-container">
-        <InputField fieldType="POSTCODE" required={true} label="Post Code" />
-        <InputField fieldType="AMOUNT" required={true} label="Yearly Income" />
-        <InputField fieldType="AMOUNT" required={true} label="Bonus" />
+        <InputField fieldType="POSTCODE" required={true} label="Postal Code"/>
+        <InputField fieldType="AMOUNT" required={true} label="Yearly Income"/>
+        <InputField fieldType="AMOUNT" required={true} label="Bonus"/>
         <DropdownField
-          label="Single/Married?"
+          label="Single/Married"
           options={["Single", "Married"]}
           required={true}
         />
       </div>
 
       <div className="secondary-details-container">
-        <div className="half-width"></div>
-        <div className="half-width">
+        <div id="results">
```

Comment:
Can't approve this for now, this has to be updated to reflect the requirements. I'll look into the actual behaviour of all of this new code during the demo.